src/routes/apiRoute.js

```diff
@@ -0,0 +1,14 @@
+import * as express from 'express';
+import routes from './';
```

Comment:
As a good practice, you can import destructured routes from `./`
`import { getProducts, getProduct, getReviews, addProduct, getUsers } from './'`
and then use them appropriately

Reply:
Agreed. Thanks for the suggestion. Updated the code.